homepairs/homepairsUI/src/modals/property-modals/EditTenantModal/EditTenantModal.tsx

```diff
@@ -38,7 +38,7 @@ function setInputStyles(){
                 paddingHorizontal: BaseStyles.MarginPadding.mediumConst,
         },
         modalContainer: {
-            flex: 1,
+            flex: Platform.OS === 'web' ? undefined : 1,
```

Comment:
you used a conditional flexGrow for the appliance modal's scrollContentContainerStyle + always flex: 1 for its modalContainer, whereas for the tenant modal you made the flex for the modal Container conditional, you sure this is what you wanted?